scripts/west_commands/build.py

```diff
@@ -24,6 +24,7 @@
 
 BUILD_USAGE = '''\
 west build [-h] [-b BOARD[@REV]]] [-d BUILD_DIR]
+           [-s SNIPPET] [-i SHIELD]
```

Comment:
isn't snippet uppercase S? then you could take the lowercase one for this

Reply:
Yeah, `-s` is used for source directory for backwards compatibility. I've fixed the capitalisation of the `-S`, thanks.